proto/asn1/common/common.h

```diff
@@ -0,0 +1,30 @@
+#ifndef COMMON_H_
+#define COMMON_H_
+
+#include <math.h>
+#include <stdint.h>
+
+#include <vector>
+
+// Returns the number of bytes needed to |base| encode |value| into a
+// variable-length unsigned integer with no leading zeros.
+uint8_t GetVariableIntLen(const uint64_t value, const size_t base);
```

Comment:
```suggestion
uint8_t GetVariableIntLen(uint64_t value, size_t base);
```

There's no need to pass these integers as const (throughout this file and this PR). Integers are always copied by value, and the 'const' doesn't really buy anything for readability or correctness :)